opm/input/eclipse/Schedule/Well/Well.hpp

```diff
@@ -104,6 +104,15 @@ class Well {
 
     using GasInflowEquation = WellGasInflowEquation;
 
+    enum WellRefinementType {
+        STANDARD,
+        LGR,
+        MIXED,
+    };
+    void flag_lgr_well(void);
+    void set_lgr_well_tag(const std::string& lgr_tag_name);
+    bool is_lgr_well(void) const;
+    const std::string& get_lgr_well_tag(void) const;
```

Comment:
We generally prefer to have empty parameter lists&ndash;e.g., `const std::string& get_lgr_well_tag() const`&ndash;instead of using `void` as the parameter type.